src/main/java/uk/gov/hmcts/reform/pip/account/management/service/SensitivityService.java

```diff
@@ -23,9 +22,10 @@ public class SensitivityService {
      * @return true if user has permission to see the publication, false if not.
      */
     public boolean checkAuthorisation(PiUser user, ListType listType, Sensitivity sensitivity) {
+        CombinedRoles combinedRoles = new CombinedRoles();
```

Comment:
If you made the class an utility class consisting of public static methods, there is no need to instantiate the object. What you need here is just getting some fixed values out. See comment above :)